package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "plexus",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "jscs": "~2.5.1",
+    "jshint": "~2.9.1-rc1"
+  },
+  "devDependencies": {
+    "jshint": "~2.9.1-rc1",
+    "jscs": "~2.5.1"
```

Comment:
If we have npm packages listed in `dependencies`, is it necessary that they also go in `devDependencies`?

I think `jscs` and `jshint` would belong in devDependencies but not dependencies, although that choice doesn't matter so much, but they shouldn't be in both. Looking at the manpage for `npm help install`, `npm install` will install any package listed in dependencies or devDependencies unless the `--production` flag is used.

https://docs.npmjs.com/files/package.json#devdependencies